### ISSUE TITLE
2.1.2: Exception if not authed when calling `slackr_ims` not handled, surfaces in `slackr_setup`

### ISSUE THREAD

**Issue by BenCarlsen** [mrkaye97/slackr#143]
The POST call `tmp <- httr::POST("https://slack.com/api/conversations.list?types=im", body = list(token = bot_user_oauth_token))` in slackr_ims can return a 200 with body 
```
$ok
[1] FALSE

$error
[1] "not_authed"
```
This is not handled, so it gets surfaced to the user as the uninformative
```
Error in if ((nrow(ims) == 0) | (nrow(users) == 0)) { : 
  argument is of length zero
```

Adding `im:write` permission to the App appeared to resolve the issue, but this is not a recommended scope listed in the vignette. The app is still unable to post with `slackr`, although `slackr_bot` works.

Additionally this version of the package throws recurring ```Error in fetch(key) :   lazy-load database '/Users/ben/Library/R/4.0/library/slackr/help/slackr.rdb' is corrupt``` errors for me, although this may be user error.

**Comment by mrkaye97** [comment]
Hey Ben,

Thanks for the issue! I'll look into fixing this error later tonight. 

FYI, `im:write` actually is a recommended scope ([see here](https://mrkaye97.github.io/slackr/articles/scoped-bot-setup.html)). 

The `Error in fetch(key) ... `  bit should be fixed with an R restart

**Comment by mrkaye97** [comment]
Should be fixed in #149. Please reopen this issue if not
